src/args.ts

```diff
@@ -0,0 +1,49 @@
+interface iCommands {
+  command: string;
+  description: string;
+}
+
+const commands: iCommands[] = [
```

Comment:
```suggestion
const commands: Command[] = [
```

(adding this suggestion here too in case you want to apply them from the GH UI)